applications/sasquatch/charts/telegraf-kafka-consumer/README.md

```diff
@@ -29,9 +29,13 @@ Telegraf is an agent written in Go for collecting, processing, aggregating, and
 | influxdb.database | string | `"telegraf-kafka-consumer-v1"` | Name of the InfluxDB v1 database to write to. |
 | influxdb2.bucket | string | `"telegraf-kafka-consumer"` | Name of the InfluxDB v2 bucket to write to. |
 | kafkaConsumers.test.enabled | bool | `false` | Enable the Telegraf Kafka consumer. |
+| kafkaConsumers.test.fields | list | `[]` | Avro fields to be treated as fields.  If not specified, anything that comes in that is not marked as a tag. |
 | kafkaConsumers.test.flush_interval | string | `"1s"` | Default data flushing interval to InfluxDB. |
 | kafkaConsumers.test.interval | string | `"1s"` | Data collection interval for the Kafka consumer. |
+| kafkaConsumers.test.tags | list | `[]` | Avro fields to be treated as tags |
+| kafkaConsumers.test.timestamp_field | string | `"private_efdStamp"` | Avro field that contains the timestamp |
```

Comment:
`Avro field to be used as the InfluxDB time (optional). If not specified, it uses the time when Telegraf received the measurement.`